bowconvert.go

```diff
@@ -130,6 +138,73 @@ func ToString(input interface{}) (output string, ok bool) {
 		return fmt.Sprintf("%f", input), true
 	case float64:
 		return fmt.Sprintf("%f", input), true
+	case arrow.Timestamp:
+		return strconv.Itoa(int(input)), true
+	}
+	return
+}
+
+// ToTimestampSec returns an arrow.Timestamp value and a bool whether the conversion was successful or not.
+// String values are first interpreted with strconv.ParseInt.
+// If it fails, the values are parsed with arrow.TimestampFromString with the arrow.Second time unit.
+func ToTimestampSec(input interface{}) (output arrow.Timestamp, ok bool) {
+	return toTimestamp(input, arrow.Second)
+}
+
+// ToTimestampMilli returns an arrow.Timestamp value and a bool whether the conversion was successful or not.
+// String values are first interpreted with strconv.ParseInt.
+// If it fails, the values are parsed with arrow.TimestampFromString with the arrow.Millisecond time unit.
+func ToTimestampMilli(input interface{}) (output arrow.Timestamp, ok bool) {
+	return toTimestamp(input, arrow.Millisecond)
+}
+
+// ToTimestampMicro returns an arrow.Timestamp value and a bool whether the conversion was successful or not.
+// String values are first interpreted with strconv.ParseInt.
+// If it fails, the values are parsed with arrow.TimestampFromString with the arrow.Microsecond time unit.
+func ToTimestampMicro(input interface{}) (output arrow.Timestamp, ok bool) {
+	return toTimestamp(input, arrow.Microsecond)
+}
+
+// ToTimestampNano returns an arrow.Timestamp value and a bool whether the conversion was successful or not.
+// String values are first interpreted with strconv.ParseInt.
+// If it fails, the values are parsed with arrow.TimestampFromString with the arrow.Nanosecond time unit.
+func ToTimestampNano(input interface{}) (output arrow.Timestamp, ok bool) {
+	return toTimestamp(input, arrow.Nanosecond)
+}
+
+func toTimestamp(input interface{}, timeUnit arrow.TimeUnit) (output arrow.Timestamp, ok bool) {
+	switch input := input.(type) {
+	case json.Number:
+		output, err := input.Int64()
+		return arrow.Timestamp(output), err == nil
+	case int:
+		return arrow.Timestamp(input), true
+	case int8:
+		return arrow.Timestamp(input), true
+	case int16:
+		return arrow.Timestamp(input), true
+	case int32:
+		return arrow.Timestamp(input), true
+	case int64:
+		return arrow.Timestamp(input), true
+	case float32:
+		return arrow.Timestamp(input), true
+	case float64:
+		return arrow.Timestamp(input), true
+	case bool:
+		if input {
+			return 1, true
+		}
+		return 0, true
+	case string:
+		output, err := strconv.ParseInt(input, 10, 64)
```

Comment:
Let's create a function to ease readability

Reply:
Done :)